_pages/documentation/general_docs/kconfig_build_system.md

```
@@ -0,0 +1,201 @@
+---
+layout: documentation
+title: Kconfig Build System
+doc: gem5 documentation
+parent: kconfig_build_system
+permalink: /documentation/general_docs/kconfig_build_system/
+---
+
+This guide is intended for advanced users who need to build GEM5 with
```

gem5 should always be lowercase

```suggestion
This guide is intended for advanced users who need to build gem5 with
```